x-pack/plugins/index_management/public/application/components/mappings_editor/__jest__/client_integration/mappings_editor.test.tsx

```diff
@@ -436,4 +436,46 @@ describe('Mappings editor: core', () => {
       expect(data).toEqual(updatedMappings);
     });
   });
+
+  describe('multi-fields support', () => {
+    it('allows multi-fields for most types', async () => {
```

Comment:
is there value in testing more than just `wildcard`?

Reply:
I don't think it adds a lot of value to test other types, since this is not checking the API but only that the button is shown on the client side